commons/passive/garmin/garmin_generic_heartrate.avsc

```diff
@@ -0,0 +1,46 @@
+{
+   "name":"GarminGenericHeartRate",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Provides the data about real time Heart rate data.",
+   "fields":[
+
+      {
+            "name":"time",
+            "type":"double",
+            "doc":"Last updated time."
+         },
+         {
+            "name":"timeReceived",
+            "type":"double",
+            "doc":"Device receiver timestamp in UTC (s)."
+         },
+
+      {
+         "name":"currentHeartRate",
+         "type":"int",
+         "doc":"Current heart rate (bpm)."
+      },
+      {
+         "name":"currentRestingHeartRate",
```

Comment:
Same here. you can remove `current` from all fields

Reply:
Done. Available in latest commit.